salt/config/__init__.py

```diff
@@ -982,6 +989,7 @@
     'transport': 'zeromq',
     'auth_timeout': 60,
     'auth_tries': 7,
+    'master_tries': 1,
```

Comment:
Why do we not want -1 here?  What happens after the minion fails to connect due to an issue with the local networking.  Does this mean if a box boots up with a bad port config,  it will require manual intervention to get it to highstate once the network is up?